comment/tests/test_template_tags.py

```diff
@@ -74,6 +75,8 @@ def test_render_comments(self):
         # no pagination
         self.assertEqual(data['comments'].count(), count)  # parent comment only
         self.assertEqual(data['login_url'], settings.LOGIN_URL)
+        # check if `request` object is passed in context template
+        self.assertIsInstance(data['request'], WSGIRequest)
```

Comment:
perhaps this should be:
```suggestion
        self.assertEqual(data['request'], request)
```